packages/app-i18n/src/admin/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -10,16 +10,24 @@ import { MenuButton } from "./index";
 import { Menu } from "./Menu";
 import { FormComponentProps } from "@webiny/ui/types";
 import { pluginsToProps } from "./pluginsToProps";
-import { Range, Descendant } from "slate";
 
 const EditorWrapper = styled("div")({
     border: "1px solid var(--mdc-theme-on-background)",
     borderRadius: 2,
     boxSizing: "border-box",
-    padding: 10
+    padding: 10,
+
+    "& > div:first-of-type": {
```

Comment:
Let's go with this approach maybe:
![image](https://user-images.githubusercontent.com/5121148/84390263-95fed880-abf7-11ea-8e52-ac94835e25f5.png)